webapp/src/widgets/valueSelector.tsx

```diff
@@ -136,44 +142,46 @@ const valueSelectorStyle = {
 
 function ValueSelector(props: Props): JSX.Element {
     return (
-        <CreatableSelect
-            captureMenuScroll={true}
-            maxMenuHeight={1200}
-            isMulti={props.isMulti}
-            isClearable={true}
-            styles={valueSelectorStyle}
-            formatOptionLabel={(option: IPropertyOption, meta: FormatOptionLabelMeta<IPropertyOption, true | false>) => (
-                <ValueSelectorLabel
-                    option={option}
-                    meta={meta}
-                    onChangeColor={props.onChangeColor}
-                    onDeleteOption={props.onDeleteOption}
-                    onDeleteValue={props.onDeleteValue}
-                />
-            )}
-            className='ValueSelector'
-            options={props.options}
-            getOptionLabel={(o: IPropertyOption) => o.value}
-            getOptionValue={(o: IPropertyOption) => o.id}
-            onChange={(value: ValueType<IPropertyOption, true | false>, action: ActionMeta<IPropertyOption>): void => {
-                if (action.action === 'select-option') {
-                    if (Array.isArray(value)) {
-                        props.onChange((value as IPropertyOption[]).map((option) => option.id))
-                    } else {
-                        props.onChange((value as IPropertyOption).id)
+        <div data-testid='creatable-selector-parent'>
```

Comment:
The `data-testid` is great whenever we strictly need that, but I think we don't need that for certain cases. For example, in general, all our components have a class, based on the component type, in this case the class `ValueSelector`, I think we can relay on that, instead of using the `data-testid`. That makes a bit more "artificial" the test cases. I think is a great solution in some cases, but here I think is not needed.